js/main.js

```diff
@@ -0,0 +1,187 @@
+// Navbar
+const menuOpen = document.getElementById('menuOpen');
+const menuClose = document.getElementById('menuClose');
+
+const showNavbar = () => {
+  const navbar = document.querySelector('.navbar__container');
+
+  if (window.matchMedia('(max-width: 768px)').matches) {
+    menuOpen.style.display = 'none';
+    menuClose.style.display = 'block';
+    document.body.style.overflow = 'hidden';
+    navbar.style.left = '0';
+  }
+};
+menuOpen.addEventListener('click', showNavbar);
+
+const hiddeNavbar = () => {
+  const navbar = document.querySelector('.navbar__container');
+
+  if (window.matchMedia('(max-width: 768px)').matches) {
+    menuClose.style.display = 'none';
+    menuOpen.style.display = 'block';
+    document.body.style.overflow = 'scroll';
+    navbar.style.left = '-100%';
+  }
+};
+menuClose.addEventListener('click', hiddeNavbar);
```

Comment:
Greeting! it appears you follow the DRY (Don't Repeat Yourself) principle in most parts. However, there are a few areas where code repetition could be reduced:

__Navbar functions__: The functions `showNavbar` and `hiddeNavbar` share a lot of common code. You could consider combining them into a single function that accepts parameters to handle the differences. This would help eliminate the duplicate code. 😄